resources/js/humhub.smartVillage.js

```diff
@@ -0,0 +1,17 @@
+humhub.module('smartVillage', function(module, require, $) {
+
+    var init = function() {
+        console.log('smartVillage module activated');
+    };
+
+    var hello = function() {
+        alert(module.text('hello')+' - '+module.config.username)
```

Comment:
Hi @donni106 
We are not using this js file currently. Its a placeholder generated by Humhub code generator tool - Gii.

We can remove the javascript code in the file. But can keep the JS file for any future Javascript we might need to add for the module.

This js file is loaded when we click on module(placed on the header of humhub).
![image](https://user-images.githubusercontent.com/103625795/165449873-2d5c57fb-5dfb-4db8-a9d7-f1441ba94187.png)
This will open user facing setting of module.


And when we click on sidebar of admin menu then configuration setting of module will be open.
![image](https://user-images.githubusercontent.com/103625795/165450240-dda97313-e1ff-4f23-9ece-6443370d2c00.png)